mslib/mswms/mpl_lsec_styles.py

```diff
@@ -43,8 +43,7 @@ def __init__(self, driver, variable="air_temperature", filetype="ml"):
         self.required_datafields = [(filetype, self.variable, None)]
         if filetype != "pl":
             self.required_datafields.insert(0, (filetype, "air_pressure", "Pa"))
-        abbreviation = "".join([text[0] for text in self.variable.split("_")])
-        self.name = f"LS_{str.upper(abbreviation)}"
+        self.name = f"LS_Default_{variable}"
```

Comment:
noting, this will change length of filename too, old one:
Linear_LS_AT-Noneit2022_02_04_00_00_00vt2022_02_05_12_00_00.png

it makes sense to use the origin variable name.

Reply:
test_wms tests have to become adapted too

Reply:
The error messages in relation to libgsl are... related?